mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/ConfirmCompositeSwapScreenV2.tsx

```diff
@@ -71,23 +79,85 @@ export function ConfirmCompositeSwapScreenV2({ route }: Props): JSX.Element {
   const dispatch = useAppDispatch();
   const logger = useLogger();
   const { getTokenPrice } = useTokenPrice();
+  const { getBestPath } = useTokenBestPath();
   const hasPendingJob = useSelector((state: RootState) =>
     hasTxQueued(state.transactionQueue)
   );
   const hasPendingBroadcastJob = useSelector((state: RootState) =>
     hasBroadcastQueued(state.ocean)
   );
+  const blockCount = useSelector((state: RootState) => state.block.count ?? 0);
+  const [totalFees, setTotalFees] = useState(new BigNumber(0));
+  // const lmTokenAmount = percentage.times(pair.totalLiquidity.token)
   const currentBroadcastJob = useSelector((state: RootState) =>
     firstTransactionSelector(state.ocean)
   );
-  const blockCount = useSelector((state: RootState) => state.block.count ?? 0);
 
   const [isSubmitting, setIsSubmitting] = useState(false);
   const [isOnPage, setIsOnPage] = useState(true);
   const isFutureSwap = futureSwap !== undefined;
 
-  const TokenAIcon = getNativeIcon(tokenA.displaySymbol);
-  const TokenBIcon = getNativeIcon(tokenB.displaySymbol);
+  const { address } = useWalletContext();
+  const addressLabel = useAddressLabel(address);
+
+  const { isLight } = useThemeContext();
+  const modalSortingSnapPoints = { ios: ["60%"], android: ["60%"] };
+
+  const {
+    bottomSheetRef,
+    containerRef,
+    expandModal,
+    dismissModal,
+    isModalDisplayed,
+  } = useBottomSheet();
+
+  // TODO: refactor into common component - used by Add/Remove Liq as well
+  const BottomSheetHeader = {
+    headerStatusBarHeight: 2,
+    headerTitle: "",
+    headerBackTitleVisible: false,
+    headerStyle: tailwind("rounded-t-xl-v2 border-b-0", {
+      "bg-mono-light-v2-100": isLight,
+      "bg-mono-dark-v2-100": !isLight,
+    }),
+    headerRight: (): JSX.Element => {
+      return (
+        <ThemedTouchableOpacityV2
+          style={tailwind("mr-5 mt-4 -mb-4")}
+          onPress={dismissModal}
+          testID="close_bottom_sheet_button"
+        >
+          <ThemedIcon iconType="Feather" name="x-circle" size={22} />
+        </ThemedTouchableOpacityV2>
+      );
+    },
+  };
+
+  const getTotalFees = async (): Promise<void> => {
+    const { estimatedReturn, estimatedReturnLessDexFees } = await getBestPath(
+      tokenA.id,
+      tokenB.id
```

Comment:
```suggestion
    const { estimatedReturn, estimatedReturnLessDexFees } = await getBestPath(
      tokenA.id === "0_unified" ? "0" : tokenA.id,
      tokenB.id === "0_unified" ? "0" : tokenB.id
```

![image](https://user-images.githubusercontent.com/13292662/188534959-686beb84-808a-4c88-955d-dca7f0675124.png)